geomesa-raster/src/main/scala/org/locationtech/geomesa/raster/util/RasterUtils.scala

```diff
@@ -145,7 +233,7 @@ object RasterUtils {
   }
 
   def createRasterStore(tableName: String) = {
-    val rs = RasterStore("user", "pass", "testInstance", "zk", tableName, "SUSA", "SUSA", true)
+    val rs = RasterStore("user", "pass", "testInstance", "zk", tableName, "", "", true)
```

Comment:
I'd move this out into a test class/object, something like RasterTestUtils should be good.